101-media-services-account-create/README.md

```diff
@@ -0,0 +1,13 @@
+# Create a Media Services Account using a template
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2F101-media-services-create%2Fazuredeploy.json" target="_blank">
```

Comment:
See the contribution guide or other doc samples for how to construct a proper README.md for the repo